src/gui/torrentcontentmodel.cpp

```diff
@@ -65,6 +65,39 @@
 
 namespace
 {
+    class FolderAdaptor
```

Comment:
Consider adding some comments to the adaptor class, something about its purpose.